### DISCUSSION TITLE
# of plots in plotnft

### DISCUSSION THREAD

**Discussion by jayhohoho2019** [Chia-Network/chia-blockchain#7477] [Q&A]
Hi, in v1.2.0 and 1.2.1, neither "chia farm summary" or "chia plotnft show" shows the # of plotnft plots in the pool.  But that info is available in the GUI under the Pool tab, and it refreshes.  Where is that info coming from? Which CLI command provides that? 

Thanks.

**Comment by jayhohoho2019** [comment]
Also, in the output of "chia plotnft show".  what does "Current state from block height" mean?  It's been a block height for me that hasn't changed.  Is that supposed to change when the Current state changes (from SELF_POOLING to POOLING e.g.?)

**Reply by SlugFiller** [reply]
This basically indicates the last write to the NFT. A write has to occur every time you switch pools (or switch to/from self pool), It's written to the NFT as a transaction, to make it publicly verifiable how long you've been farming on behalf of the pool. Since farming is done towards the NFT, the pool needs verification that any successfully farmed block would be passed to it, and not left in the hands of the farmer or given to a different pool.

Each pool also has a minimal amount of blocks that have to pass before you can switch from them (100 by default), to prevent from suddenly switching pools (or going solo) immediately upon successfully farming a block, "stealing" the block in the process.

In other words, this detail shows how long you've been in the current pool, and which of the blocks that you farmed count towards that pool (blocks farmed below that height do not count).

If you have yet to change pools or join any pool, it would simply show the pool NFT's creation date (in the form of a block height).

**Comment by Proulx-S** [comment]
Same question here, or more precisely: how to get the number of plots associated with a nft from the command line?

I did not even install the gui and it would be great if I could continue avoiding it.

**Comment by alexwwstx** [comment]
I don't think there's a cmd supporting it yet

**Reply by Proulx-S** [reply]
I just found out that this is apparently resolved in this (merged) pull requested
https://github.com/Chia-Network/chia-blockchain/pull/7481

**Reply by Proulx-S** [reply]
But after updating to the latest chia version, my `chia plotnft show` still does not show plot count...

**Reply by Proulx-S** [reply]
oh, now it does
